src/utilities/pluralize.ts

```diff
@@ -6,5 +6,6 @@
  */
 export default function pluralize(word: string, count: number = 1): string {
   if (!word) return ''
+  if (word.lastIndexOf('s') === word.length - 1) return word
```

Comment:
not really part of the pagination component, but I added a small verification to make sure we don't duplicate the `s` character at the end of the string